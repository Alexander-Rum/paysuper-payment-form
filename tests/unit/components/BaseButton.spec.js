import { shallowMount } from '@vue/test-utils';
import BaseButton from '@/components/BaseButton.vue';

describe('BaseButton.vue', () => {
  it('should render content', () => {
    const wrapper = shallowMount(BaseButton, {
      slots: {
        default: 'check',
      },
    });
    expect(wrapper.text()).toEqual('check');
    expect(wrapper.element).toMatchSnapshot();
  });
});
