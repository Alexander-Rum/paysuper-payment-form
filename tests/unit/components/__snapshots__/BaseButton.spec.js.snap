// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseButton.vue should render content 1`] = `
<button
  class="base-button _color-yellow _size-default"
  type="button"
>
  <!---->
   check
</button>
`;
